src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -146,6 +147,30 @@ public function isWritable($class, $property, array $context = array())
         return null !== $reflectionMethod;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function isInitializable(string $class, string $property, array $context = array()): ?bool
+    {
+        try {
+            $reflectionClass = new \ReflectionClass($class);
+        } catch (\ReflectionException $e) {
+            return null;
+        }
+
+        if ($constructor = $reflectionClass->getConstructor()) {
```

Comment:
Why don't you test if the constructor is instantiable first?

Reply:
Good idea, I'll update the code.

Reply:
done